wordpress-persistent-disks/mysql.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: mysql
+  labels:
+    app: mysql
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mysql
+  template:
+    metadata:
+      labels:
+        app: mysql
+    spec:
+      containers:
+        - image: mysql:5.6
+          name: mysql
+          env:
+            - name: MYSQL_ROOT_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: mysql
+                  key: password
```

Comment:
@mike-trewartha for now please follow the https://cloud.google.com/container-engine/docs/tutorials/persistent-disk/. The new version of the tutorial (which this PR adds) is not published yet.